Src/Sound/PGESound.cpp

```diff
@@ -0,0 +1,28 @@
+// The PGE objects are simple data containers for their referenced material
+#include "PGESound.h"
+
+#include <String/String.h>
+#include <Misc/FilePath.h>
+
+#include "Sound.h"
+#include "SoundStream.h"
+
+
+
+namespace PGE {
```

Comment:
The PGE namespace is not to be extended in CBN.

Reply:
I knew a PGE module was going to be required, but I did not know exactly what for.
Seems like its use will be to manage sdl_audio.
I'm not sure what that's going to look like.
Everything in Sound/ is early testing to get a sound to play.